lib/app/routes/exercises.rb

```diff
@@ -133,6 +134,25 @@ class Exercises < Core
         Hack::UpdatesUserExercise.new(submission.user_id, submission.track_id, submission.slug).update
         redirect "/"
       end
+
+      get '/submissions/:key/blobs/:sha' do
+        content_type :json
+        submission = Submission.find_by_key(params[:key])
+        begin
+          blob = Octokit.blob("#{submission.user.username}/#{submission.slug}", 
+                               params[:sha])
+          src_classifer = SourceClassifier.new(File.join(File.dirname(__FILE__), "../../../bin/", 
+                                               "trainer.bin"))
+          result =  Base64.decode64(blob.content)
+          source_language = src_classifer.identify(result)
+          marked_content = ConvertsMarkdownToHTML.convert("```#{source_language.downcase}\n
+                                                            #{result}\n```")
+          content = { data: marked_content } 
+          content.to_json
+        rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?